src/node.js

```diff
@@ -159,8 +159,26 @@
   startup.globalVariables = function() {
     global.process = process;
     global.global = global;
-    global.GLOBAL = global;
-    global.root = global;
+
+    // Deprecate GLOBAL and root
+    ['GLOBAL', 'root'].forEach(function(name) {
+      // getter
+      const get = NativeModule.require('util').deprecate(function() {
+        return this;
+      }, "'" + name + "' is deprecated, use 'global'");
+      // setter
+      const set = function(value) {
+        const desc = {configurable: true, enumerable: true, value: value};
```

Comment:
I still think it should be writable here. Otherwise setting the property more than once would fail silently:
```js
GLOBAL = 1;
GLOBAL = 2;
console.log(GLOBAL); // prints 1
```